docs/operations/data_migration.md

```diff
@@ -586,3 +586,47 @@ Example: `cf ssh getgov-za`
 |   | Parameter                  | Description                                                                 |
 |:-:|:-------------------------- |:----------------------------------------------------------------------------|
 | 1 | **debug**                  | Increases logging detail. Defaults to False.                                |
+
+
+## Populate First Ready
+This section outlines how to run the `populate_organization_type` script.
+
+### Running on sandboxes
+
+#### Step 1: Login to CloudFoundry
+```cf login -a api.fr.cloud.gov --sso```
+
+#### Step 2: SSH into your environment
+```cf ssh getgov-{space}```
+
+Example: `cf ssh getgov-za`
+
+#### Step 3: Create a shell instance
+```/tmp/lifecycle/shell```
+
+#### Step 4: Running the script
+```./manage.py populate_organization_type {domain_election_office_filename} --debug```
```

Comment:
Can you please share the link where the csv with this full list is so people can use it when testing? Also, please add in these steps how to get the csv onto the given sandbox (as I'm sure we don't want to add it to the repo).